content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,173 @@
+
+To enable a shipping method, navigate to :guilabel:`Inventory app --> Configuration --> Settings`,
+and click the :guilabel:`FedEx Shipping Methods` button below the :guilabel:`FedEx Connector`
+checkbox. Doing so opens a page that shows all existing FedEx shipping methods.
+
+.. note::
+   To see all shipping methods for every shipper with a connector enabled, navigate to
+   :menuselection:`Inventory app --> Configuration --> Shipping Methods`.
+
+Select a shipping method to open its form. Alternatively, click :guilabel:`New` to open a blank
+form, and configure a new shipping method.
+
+.. image:: fedex/fedex-form.png
+   :align: center
+   :alt: The form for a FedEx shipping method.
+
+.. important::
+   Enabling the FedEx shipping connector automatically creates two default shipping methods: *FedEx
+   US* and *FedEx International*. Each of these methods are pre-configured with test credentials,
+   allowing them to be used for testing purposes.
+
+   Before the shipping method can be used to create actual shipments, the test credentials must be
+   replaced with credentials from a valid FedEx account.
+
+General information
+-------------------
+
+At the very top of a shipping method form are a number of fields used to configure the way the
+method operates in Odoo. In the :guilabel:`Provider` field, select :guilabel:`FedEx` from the
+drop-down menu, if it is not already selected.
+
+The rest of the fields in this section are general to all shipping providers. For details on how to
+fill them out, see the documentation on :doc:`third-party shippers <third_party_shipper>`.
+
+Fedex Configuration
+-------------------
+
+The options in the :guilabel:`Fedex Configuration` tab of a FedEx shipping method are used to
+connect the method to a FedEx account, and configure the shipping details associated with the method
+(drop-off type, package type, etc.).
+
+A FedEx business account is required to obtain the information needed to fill out the fields in this
+tab. To create a new account, navigate to FedEx's `Open Account
+<https://www.fedex.com/en-us/open-account.html>` page, click on :guilabel:`Create Account`, and
+follow the instructions.
+
+Password and Account Number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A *password* is used, along with a username, to log into a FedEx account. An *account number* is the
+unique number assigned to each FedEx account.
+
+To find a FedEx account number, log in to a FedEx account at https://www.fedex.com. Click on the
+account holder's name in the top-right corner of the screen, and select :menuselection:`My Profile`
+from the drop-down menu.
+
+On the profile page, click :guilabel:`Account Management` on the left side of the screen. The
+account number is displayed on this screen.
+
+Once the password and account number are determined, enter them in the :guilabel:`Password` and
+:guilabel:`Account Number` fields on the :guilabel:`Fedex Configuration` tab of the shipping method.
+
+Developer Key and Meter Number
```

Comment:
you may want to consider putting this above the 'Password/account number' portion because the reader will see the 'Developer Key' field first, and might wonder why it isn't being discussed first. Just something to consider...